### PR TITLE
doc: fix desc of option "mon cluster log file"

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
@baruza in case you are interested.

File: doc/rados/troubleshooting/log-and-debug.rst [hunk 1]
```diff
@@ -396,10 +396,17 @@ settings:
 
 ``mon cluster log file``
 
-:Description: The location of the cluster's log file. 
+:Description: The locations of the cluster's log files. There are two channels in
+              Ceph: ``cluster`` and ``audit``. This option represents a mapping
+              rom channels to log files, where the log entries of that
```
Comment (jcsp) [inline doc/rados/troubleshooting/log-and-debug.rst]: s/rom/from/

File: doc/rados/troubleshooting/log-and-debug.rst [hunk 2]
```diff
@@ -396,10 +396,17 @@ settings:
...
+              channel are sent to. And the ``default`` entry is a fallback
```
Comment (jcsp) [inline doc/rados/troubleshooting/log-and-debug.rst]: s/And the/The/

**Comment by tchaikov** [comment]
@jcsp fixed and repushed.

**Pull request merged**